rpcd/etc/openstack_deploy/user_rpco_variables_defaults.yml

```diff
@@ -1,5 +1,5 @@
 ---
-# Copyright 2014, Rackspace US, Inc.
+# Copyright 2015, Rackspace US, Inc.
```

Comment:
ya, I noticed that last time I amended but forgot to change.  I think it came from rpcd/etc/openstack_deploy/user_variables.yml though, where it was 2015